wolfcrypt/src/random.c

```diff
@@ -2455,7 +2455,7 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
     }
 
 #elif defined(WOLFSSL_ESPIDF)
-    #if defined(WOLFSSL_ESPWROOM32) || defined(WOLFSSL_ESPWROOM32SE)
+    #if defined(WOLFSSL_ESPWROOM32) || defined(WOLFSSL_ESPWROOM32SE) || defined(WOLFSSL_ESP8266)
```

Comment:
Is `WOLFSSL_ESP8266` generic enough? It looks like `ESP8266` is a line of ESP32's which include the esp_random support, but are there other ones that could possibly benefit also? Would maybe `WOLFSSL_ESP32` be too generic?

Also please limit line length to 80 characters. Thanks

Reply:
Good point. I broke out the ESP8266 into a separate section to be more conspicuously different and not commingled with ESP32 code.